### ISSUE TITLE
Customer support messaging for perf

### ISSUE THREAD

**Issue by jonbell-lot23** [RecordReplay/devtools#3694]

**Comment by jasonLaster** [comment]
Here are some quick thoughts

1. we should have a feedback button in the bottom left
2. it would be have an alert that shows up in the bottom left if there is something slow... maybe it goes away at a certain point. it should be lightweight...
3. we should have a perf panel that help users see high level perf details and slow requests

<img width="1881" alt="feedback" src="https://user-images.githubusercontent.com/254562/134255674-209acfad-e7a1-41aa-ba93-242802713219.png">

**Comment by jonbell-lot23** [comment]
Dig it, thanks!

**Comment by jonbell-lot23** [comment]
Copied from #3676 

Jon to mock up visuals for how to show perf issues. Things to consider:

  Loading screen if it's taking too long
  Session failed (perhaps with an option to set Replay to public?)
  Look at how React DevTools does the bug report

**Comment by jasonLaster** [comment]
@jonbell-lot23 @jaril Here's an idea. Can we do this with intercom's messenger API? I'd love to drive more conversations / support through the intercom chat. What if we had a little chat popup say something like "Uh oh, looks like X was slow. Let us know if this got in your way, we'd love to help" 



<img width="423" alt="Screen Shot 2021-10-12 at 11 46 25 AM" src="https://user-images.githubusercontent.com/254562/137012225-57370545-42a8-4db8-9053-e0ca8a6f23a5.png">

**Comment by jasonLaster** [comment]
Yeah totally - i think its just a way for us to say we saw it was slow and are on it...
